dynamodb/toddl.go

```diff
@@ -28,18 +28,18 @@ import (
 // Spanner. It uses the source schema in conv.SrcSchema, and writes
 // the Spanner schema to conv.SpSchema.
 func schemaToDDL(conv *internal.Conv) error {
-	// Tracks Spanner names that have been used for indexes. We use this to ensure we generate unique names when
-	// we map from DynamoDB to Spanner since we want Spanner table names and index names to be distinct.
-	usedNames := make(map[string]bool)
-	// We need to pre-populate usedNames with Spanner table names to handle collisions
-	// with table names, foreign key names and index names.
+	// conv.UsedNames tracks Spanner names that have been used for table names, foreign key constraints
+	// and indexes. We use this to ensure we generate unique names when
+	// we map from MySQL to Spanner since Spanner requires all these names to be
+	// distinct and should not differ only in case.
+	// We need to pre-populate conv.UsedNames with Spanner table
+	// names to handle collision with foreign key names and index names.
```

Comment:
I wonder if this comment makes sense here anymore? How name collisions are handled and usedNames are tracked are transparent to this function, and that seem to be by design. 

I would expect readers to look at mapping.go to understand how userNames are handled. Thus, suggest that we move some of the text there.